src/components/PassphraseBox.css

```diff
@@ -59,81 +64,98 @@
     pointer-events: all;
 }
 
-.passphrase-box .prompt {
-    transition: opacity 300ms;
-}
-
-.passphrase-box.input-valid .prompt,
-.passphrase-box.hide-input .prompt {
-    opacity: 0;
-}
-
 .passphrase-box.hide-cancel .cancel {
     display:none;
 }
 
-/* S E T T E R */
+.passphrase-box .prompt {
+    margin-top: -2.75rem;
+    line-height: 1;
+    pointer-events: none;
+}
 
+/* S E T T E R */
+.passphrase-box .repeat-password,
 .passphrase-box .password-strength {
-    margin-top: -1.75rem;
-    transform: translate3d(0, -6.75rem, 0);
+    margin-top: -2.75rem;
+    line-height: 1;
     opacity: 0;
     transition: opacity 150ms;
     pointer-events: none;
 }
 
+.passphrase-box:not(.input-valid).strength-0 .password-strength.strength-0,
 .passphrase-box.input-valid.strength-8 .password-strength.strength-8,
 .passphrase-box.input-valid.strength-10 .password-strength.strength-10,
 .passphrase-box.input-valid.strength-12 .password-strength.strength-12 {
     opacity: 1;
     transition-delay: 150ms;
 }
 
-.passphrase-box .password-hint,
 .passphrase-box .password-skip {
     transition: opacity 300ms;
 }
 
-.passphrase-box .password-hint.shake{
+.passphrase-box .password-strength.shake{
     color: var(--nimiq-red);
 }
 
-.passphrase-box.input-valid .password-hint,
 .passphrase-box.input-valid .password-skip {
     opacity: 0;
     pointer-events: none;
 }
 
-.passphrase-box .password-hint {
-    margin-top: -0.625rem;
-    margin-bottom: 0.5rem;
-}
-
 .passphrase-box .password-skip {
-    opacity: 0.7;
-    text-decoration: underline;
     cursor: pointer;
+    display: block;
+    font-size: 1.625rem;
+    transform: scale(1);
+    transition: transform .45s ease;
+}
+
+.passphrase-box .password-skip i.nq-icon {
+    background-image: url('data:image/svg+xml,<svg xmlns="http://www.w3.org/2000/svg" x="0px" y="0px" color="%23ffffff" viewBox="0 0 10 17"><polygon fill="currentColor" points="1.43,17 0,15.58 7.13,8.5 0,1.42 1.43,0 10,8.5 "/></svg>');
+    margin-left: 0.5rem;
+    height: 0.875rem;
+    width: 0.875rem;
+    transition: transform .3s ease;
+}
+
+.password-skip:hover,
+.password-skip:focus {
+  transform: scale(1.03);
```

Comment:
I think we should remove scale transforms, especially those small ones, as they just look blurry.